tests/generic_inline_admin/tests.py

```diff
@@ -72,24 +72,58 @@ def test_basic_add_POST(self):
 
     def test_basic_edit_POST(self):
         """
-        A smoke test to ensure POST on edit_view works.
+        A test to ensure POST on edit_view saves changes to the database.
```

Comment:
I think the changes to this file could be omitted. At least the cosmetic cleanups shouldn't be included. There are a number of docstrings that say, "A smoke test" which could be cleaned up together.